.github/workflows/cmake.yml

```diff
@@ -1,86 +0,0 @@
-name: CMake
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      # Don't cancel the macOS build if the Linux build fails, etc.
```

Comment:
I removed all the comments as the steps must be clear enough to understand what's happening by themselves.